cabal-install/tests/UnitTests/Distribution/Solver/Modular/Solver.hs

```diff
@@ -1301,8 +1295,6 @@ dbBase =
 dbNonupgrade :: ExampleDb
 dbNonupgrade =
   [ Left $ exInst "ghc" 1 "ghc-1" []
-  , Left $ exInst "ghci" 1 "ghci-1" []
-  , Left $ exInst "ghc-boot" 1 "ghc-boot-1" []
   , Right $ exAv "ghc" 2 []
   , Right $ exAv "ghci" 2 []
   , Right $ exAv "ghc-boot" 2 []
```

Comment:
The source versions of ghci and ghc-boot and packages B and C should also me removed.